translations/messages+intl-icu.de.yml

```
@@ -10,6 +10,7 @@ access.token: "Zugangscode"
 access.token.send: "Zugangscode versenden"
 access.token.copied: "Zugangscode wurde kopiert."
 accessibility: "Barrierefreiheit"
+accessibility.barrier.report: "Barriere melden"
```

According to our [conventions for composing translation keys](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/best-practices/translation-keys.html#use-dots-and-snake-case) i'd suggest to name this so:
```suggestion
accessibility.report_barrier: "Barriere melden"
```

---

done here https://github.com/demos-europe/demosplan-core/pull/2129/commits/8b6338fa439454a2b102e419cf5cdf131aaf4d00